### PR TITLE
Fix error date.

### LINKED ISSUE #1099: xbrl28:Error facts don't format date correctly.

### PR DESCRIPTION
**Pull request by ghislainfourny**
Fixes #1099

### CODE REVIEW

**Comment by dknochen** [comment]
:+1: could we add the case you've found as a test?

**Comment by ghislainfourny** [comment]
Not really... it popped up in the Columns branch because of another bug...

**Comment by dknochen** [comment]
ok, then :)

**Comment by ghislainfourny** [comment]
I don't even know how it can have worked in the first place...

**Comment by ghislainfourny** [comment]
Ah, I understand. Since the period didn't cast as a date before, it was just ignored by the zero rule. By fixing the date, we triggered the other error.

File: data/reports/secxbrl/fac.json [hunk 1]
```diff
@@ -2987,7 +2987,7 @@
...
...\r\nelse \"Cell collision with consistent values for concept Revenues.\"\r\nelse ())\r\nlet $Revenues as object? := $Revenues[1]\r\n\r\nlet $_unit := ($facts.$facts:ASPECTS.$facts:UNIT)[1]\r\nreturn\r\nswitch (true)\r\ncase (exists($TradingSymbol)) return $TradingSymbol\r\ncase (exists(($BalanceSheetDate, $Revenues))) return\r\nlet $entityID as string? := (($BalanceSheetDate, $Revenues).Aspects.\"xbrl:Entity\")[1]\r\nlet $entity as object? := entities:entities($entityID)\r\nlet $computed-value := $entity.Profiles.SEC.Tickers[[1]]\r\nlet $audit-trail-message := \"fac:TradingSymbol['\" || $computed-value || \"'] = entity(\" || $entityID || \").Profiles.SEC.Tickers\"\r\nlet $audit-trail-message as string* := ($audit-trail-message, $warnings)\r\nlet $source-facts as object* := ($BalanceSheetDate, $Revenues, $entity)\r\nreturn\r\ncopy $newfact :=\r\nrules:create-computed-fact( ($BalanceSheetDate, $Revenues)[1],\r\n\"fac:TradingSymbol\",\r\nif (exists($computed-value)) then $computed-value else \"N/A\",\r\n$rule,\r\n$audit-trail-message,\r\n$source-facts,\r\n$options)\r\nmodify ( replace value of json $newfact(\"Type\") with \"NonNumericValue\" )\r\nreturn $newfact\r\ndefault return {}",
```
Comment (ghislainfourny) [inline data/reports/secxbrl/fac.json]: The entities module import uses the old URI with xbrl.io...

**Pull request merged**